packages/flutter/test/widgets/window.dart

```diff
@@ -6,172 +6,200 @@ import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+Future<Object?>? Function(MethodCall)? _createWindowMethodCallHandler(WidgetTester tester) {
+  return (MethodCall call) async {
+    final Map<Object?, Object?> args = call.arguments as Map<Object?, Object?>;
+    if (call.method == 'createWindow') {
+      final List<Object?> size = args['size']! as List<Object?>;
+
+      await tester.binding.defaultBinaryMessenger.handlePlatformMessage(
+        SystemChannels.windowing.name,
+        SystemChannels.windowing.codec.encodeMethodCall(
+          MethodCall('onWindowCreated', {'viewId': tester.view.viewId, 'parentViewId': null}),
```

Comment:
Adding a type annotation:
```suggestion
          MethodCall('onWindowCreated', <String, Object?>{'viewId': tester.view.viewId, 'parentViewId': null}),
```